src/main/java/invtweaks/InvTweaksConfigManager.java

```diff
@@ -215,8 +217,9 @@ private boolean loadConfig() {
         }
     }
 
-    private boolean extractFile(ResourceLocation resource, File destination) {
-        try(InputStream input = mc.getResourceManager().getResource(resource).getInputStream()) {
+    private boolean extractFile(String resourcePath, File destination) {
+        ClassLoader classloader = Thread.currentThread().getContextClassLoader();
+        try (InputStream input = classloader.getResourceAsStream(resourcePath)) {
```

Comment:
I managed to get things into a debugger, apparently someone decided it'd be a bright idea to do a case-sensitive check on the filename... while the names are all forced to lowercase by the ResourceLocation constructor.